test/Driver/verbose.swift

```diff
@@ -0,0 +1,4 @@
+// RUN: %swiftc_driver -driver-print-jobs -target x86_64-unknown-linux-gnu -v %s 2>&1 | %FileCheck %s -check-prefix=VERBOSE_CLANG
+// RUN: %swiftc_driver -driver-print-jobs -target x86_64-unknown-windows-misc -v %s 2>&1 | %FileCheck %s -check-prefix=VERBOSE_CLANG
```

Comment:
"misc"? Is this correct? (It probably doesn't matter for that part of the triple, but…)

Reply:
That was probably a typo that I need to fix. That said, should it be "msvc" or "cygnus"?

Reply:
"msvc", I think. I would expect the Cygnus build to use the GenericUnix toolchain. (Actually, there's a Cygnus toolchain that extends GenericUnix, but in this case it'll be the same.)

Reply:
Done.